introc/crash_course_introduction_to_c.tex

```diff
@@ -1,6 +1,7 @@
 \section{Crash course introduction to C}
 
-The only way to start learning C is by starting with the \emph{hello world} program. As per the original example that Kernighan and Ritchie proposed way back when, this program hasn't changed that much.
+The only way to start learning C is by starting with the \emph{hello world} program.
```

Comment:
This is simply false. How about changing "only" to "canonical" or such?

Reply:
(It was meant to be in good jest, I don't think we'll get complaints :D)